lib/util/utf8.ml

```diff
@@ -26,6 +26,15 @@ let get s i =
     else loop (next s i) (pred k)
   in loop 0 i
 
+(** [Utf8.set s n] returns the number of characters (not bytes) before index n
+    in string [s].
+*)
+let set s n =
```

Comment:
I don't understand the name of the function. `set` is likely to be either a set, or a set as in maths.

Reply:
I admis I was not happy with this name (similar to the get above). Renamed to nb_utf8_char.